apex-checks/src/test/resources/checks/testMethodInTestClassCorrect.cls

```diff
@@ -0,0 +1,12 @@
+@isTest
+public class SomeTestClass {
+	
+	public static testMethod String aTestMethod() {
+		return 'x';
```

Comment:
@JimmyRojas Yes I understand what you're saying, the thing was I thought about making the condition for a testMethod to be both things, being in a @isTest class and having an assertion, but I preferred to keep it separated because that way each check could point more specifically to where the observation is, and also in case both cases cause the check to trigger the message would be either repeated or incomplete. Anyway, there's another check that checks the assertions, I think working with both together will cover these cases better.